include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -82,6 +83,7 @@ template
     typename FormulaPolicy = strategy::andoyer,
     typename Spheroid = srs::spheroid<double>,
     typename CalculationType = void,
+    bool Bisection = false,
```

Comment:
If it's only added for testing purposes should it be a part of the official interface? You could e.g. implement a version of this strategy in details and outside expose the one without additional parameters.

Is this also the case for the `EnableClosestPoint` parameter below?

Reply:
>If it's only added for testing purposes should it be a part of the interface? You could e.g. implement a version of this strategy in details and outside expose the one without additional parameters.

I exposed it in the interface since I think it could be potentially useful for some users (in some cases gives more accurate results; but taking more time). But if you insist I can hide it, I do not have a strong opinion since I do not research the variety of cases that bisection method is more accurate than the default.

> Is this also the case for the EnableClosestPoint parameter below?

This is going to be used by a new algorithm that returns the closest point, but since I didn't decide on the interface yet I can remove it for now.

Reply:
I think that if `Bisection` parameter is an important parameter which will be used often by the users it should probably be made the second one (like `SeriesOrder` in other strategies). Though it may be too late for that due to backward-compatibility issues so the reasonable approach is to leave it where it is now.

However if it's not that important as you say and used only for testing then I think it should be removed and only exposed in some strategy defined in details which you could use in tests. If some user was interested it could be made public in any time.

With that said this is not a strong opinion so I'm ok with one way or the other.

Reply:
I hide the `Bisection` method into `strategy::distance::detail` namespace.